ToDoList/App.js

```diff
@@ -1,22 +1,63 @@
 import React, { Component } from 'react';
-import { StyleSheet, View } from 'react-native';
+import { StyleSheet, View, ScrollView } from 'react-native';
+// import GenerateForm from 'react-native-form-builder';
+
+
 
 import Title from './components/title/title';
 import ToDo from './components/todo/todo';
+import AddToDo from './components/form/form';
 
 
 export default class App extends Component {
+  constructor(props) {
+    super(props);
+    this.addItem = this.addItem.bind(this);
+    this.forcingUpdate = this.forcingUpdate.bind(this);
+
+    this.state = {
+      todos: ['Do Laundry', 'Wash Dishes']
+    }
+  }
+
+  addItem(newToDo) {
+    console.log('adding... ', newToDo);
```

Comment:
Console.log again if you use vscode I usually will do a search for console.log to remove them before I push.